### DISCUSSION TITLE
Teleportation potion not allowed :(

### DISCUSSION THREAD

**Discussion by TostadoraZD** [Pryaxis/TShock#3021] [Q&A]
I know that to solve the problem that Tshock does not allow players to use Rod of Discord you have to write the command "group addperm guest tshock.tp.override" and "group addperm guest tshock.tp.rod" (instead of "guest" It could be another group), but I don't know how to solve the problem with the "Teleportation potion" (the purple potion that teleports you to a random place), when someone uses that potion Tshock doesn't allow it. If anyone knew how to solve it I would greatly appreciate it.

PS: This is my first time asking for help here on GitHub, sorry if I posted this in the wrong place.

**Comment by sgkoishi** [comment]
https://github.com/Pryaxis/TShock/blob/general-devel/docs%2Fpermission-descriptions.md#L529-L531
Short answer: `tshock.tp.tppotion`

**Reply by TostadoraZD** [reply]
THANK YOU!

**Comment by zyrafaq** [comment]
tshock.tp.tppotion
